src/tests/adding.md

```diff
@@ -291,6 +294,37 @@ fn main() {
 //~| ERROR this pattern has 1 field, but the corresponding tuple struct has 3 fields [E0023]
 ```
 
+#### When error line cannot be specified
+
+Let's this about this test:
+
+```rust,ignore
+fn main() {
+    let a: *const [_] = &[1, 2, 3];
+    unsafe {
+        let _b = (*a)[3];
+    }
+}
+```
+
+We want to ensure this shows "index out of bounds" but we cannot use the `ERROR` annotation
```

Comment:
Maybe specify "at runtime" to explain why this has the `run-fail`? I'm not sure this is really necessary though.
```suggestion
We want to ensure this shows "index out of bounds" at runtime, but we cannot use the `ERROR` annotation
```

Reply:
Ah, copied/pasted headers from the original test but `run-fail` may be confusing. Just removed it.